src/form.jsx

```diff
@@ -49,6 +52,15 @@ export class Form extends React.Component {
 
   static mapDispatchToProps(dispatch, ownProps) {
     return {
+      updateVisibleFieldsArr: (formId=ownProps.id, fieldName) => {
```

Comment:
Since this is expecting array, the second parameter should be plural.

Reply:
83a3255